app/server.js

```diff
@@ -1,13 +1,28 @@
-require('./insights').setup()
 const Hapi = require('@hapi/hapi')
+const config = require('./config')
+const catbox = config.useRedis ? require('@hapi/catbox-redis') : require('@hapi/catbox-memory')
+const catboxOptions = config.useRedis ? config.catboxOptions : {}
```

Comment:
We could determine the catbox config in the `config` module when we're setting it up to begin with?